index.d.ts

```diff
@@ -0,0 +1,33 @@
+interface ClearablePromise<T> extends Promise<T> {
+	/**
+	 * Clears the delay and settles the promise.
+	 *
+	 * @memberof ClearablePromise
```

Comment:
I'm not that familiar with TS documentation, but isn't this moot as it's already clear from the context and TS should be able to infer it itself?